Description: Work around the maven-repo-helper bug #802907. When two plugins have
 dependencies, if the first plugin is ignored the dependencies of the second
 plugin are replaced by the ones of the first plugin. Here the maven-jdocbook-plugin
 dependency (hibernate-jdocbook-style) replaces the maven-processor-plugin
 dependency (jboss-logging-processor) and breaks the build.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -593,19 +593,6 @@
                     <artifactId>maven-jdocbook-plugin</artifactId>
                     <version>2.3.10</version>
                     <extensions>true</extensions>
-                    <dependencies>
-                        <dependency>
-                            <groupId>org.jboss.pressgang</groupId>
-                            <artifactId>pressgang-xslt-ns</artifactId>
-                            <version>3.1.4</version>
-                        </dependency>
-                        <dependency>
-                            <groupId>org.hibernate</groupId>
-                            <artifactId>hibernate-jdocbook-style</artifactId>
-                            <version>3.0.2</version>
-                            <type>jdocbook-style</type>
-                        </dependency>
-                    </dependencies>
                 </plugin>
                 <plugin>
                     <groupId>org.asciidoctor</groupId>
